pkg/skr/gcpnfsvolumerestore/findRestoreOperation.go

```diff
@@ -0,0 +1,66 @@
+package gcpnfsvolumerestore
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/kyma-project/cloud-manager/pkg/util"
+	"google.golang.org/api/googleapi"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kyma-project/cloud-manager/api/cloud-resources/v1beta1"
+	"github.com/kyma-project/cloud-manager/pkg/composed"
+)
+
+func findRestoreOperation(ctx context.Context, st composed.State) (error, context.Context) {
+	state := st.(*State)
+	logger := composed.LoggerFromCtx(ctx)
+
+	//If deleting, continue with next steps.
+	if composed.MarkedForDeletionPredicate(ctx, st) {
+		return nil, nil
+	}
+
+	restore := state.ObjAsGcpNfsVolumeRestore()
+	opName := restore.Status.OpIdentifier
+	logger.WithValues("Nfs Restore source:", restore.Spec.Source.Backup.ToNamespacedName(state.Obj().GetNamespace()),
+		"destination:", restore.Spec.Destination.Volume.ToNamespacedName(state.Obj().GetNamespace())).Info("Finding GCP Restore Operations")
```

Comment:
```suggestion
	logger.WithValues("nfsRestoreSource", restore.Spec.Source.Backup.ToNamespacedName(state.Obj().GetNamespace()),
		"destination", restore.Spec.Destination.Volume.ToNamespacedName(state.Obj().GetNamespace())).Info("Finding GCP Restore Operations")
```
`:` will be added automatically, and our log aggregator (gcp logging) prefers if we use single words for keys.

Reply:
done. I did the same in other actions as well.